db/migrate/20210504150201_add_delivery_partner_to_csv_upload.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddDeliveryPartnerToCsvUpload < ActiveRecord::Migration[6.1]
+  # rubocop:disable Rails/NotNullColumn
+  def change
+    add_reference :partnership_csv_uploads, :delivery_partner, index: true, null: false
+  end
+  # rubocop:enable Rails/NotNullColumn
+end
```

Comment:
We are adding the migration creating `partnership_csv_uploads` in the same PR (`CreatePartnershipCsvUpload`). Can we define the `delivery_partner` field there, rather than have a new migration that requires us to skip rubocop?